packages/kube-client/lib/resources/GardenerAuthentication.js

```diff
@@ -0,0 +1,23 @@
+//
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const { mix } = require('mixwith')
+
+const { GardenerAuthentication } = require('../groups')
+
+class AdminKubeconfigRequest extends mix(GardenerAuthentication).with() {
```

Comment:
Is this only to have it in Resources?

Reply:
I would hardcode it in the `resources/index.js` 
```js
exports.Resources = _.reduce(resourceGroups, getResourceGroupMetadata, {
  AdminKubeconfigRequest: {
    kind: AdminKubeconfigRequest,
    apiVersion: 'authentication.gardener.cloud/v1alpha1'
  }
})
```

Reply:
✅